rest-api/core-rest/src/test/java/uk/gov/gchq/gaffer/rest/service/v2/OperationServiceV2IT.java

```diff
@@ -132,6 +132,7 @@ public void shouldReturnAllOperationsAsOperationDetails() throws IOException, Cl
 
         // When
         final Response response = ((RestApiV2TestClient) client).getAllOperationsAsOperationDetails();
+        assertThat(response.getStatus()).withFailMessage("client responded with an error.").isEqualTo(200);
```

Comment:
The changes here are not needed when https://github.com/gchq/Gaffer/pull/2756 is merged